### PR TITLE
Cookstyle and small fixes for style

### PR DESCRIPTION
**Pull request by damacus**
### Description

Cookstyle and small fixes for style

### Issues Resolved

None

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -13,6 +13,4 @@
 supports 'centos'
 supports 'amazon'
 supports 'opensuseleap'
-
-depends 'build-essential', '>= 8.0.1'
```
Comment (tas50) [inline metadata.rb]: If you take this out you need chef 14+

**Comment by Xorima** [comment]
Hey @damacus 

Can you look into fixing the tests, seems to be something around resources missing names?

Thanks mate

**Comment by Xorima** [comment]
@damacus I cannot work out if these are flaky tests or not here, can you run them locally and confirm?

Thanks, otherwise it looks good!

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,11 @@ This file is used to list changes made in each version of the haproxy cookbook.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## unreleased
+
+- Removed `depends_on` build-essential, as this is now in Chef Core
```
Comment (tas50) [inline CHANGELOG.md]: 💯

**Pull request merged**